car_wash/settings.py

```diff
@@ -23,28 +23,33 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-x^av-oc%u-0$fu)c+m4=#yo-m#wcrb$uj#)+n#746job@=yr&s"
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv('DEBUG')
 
 ALLOWED_HOSTS = [
-    "10.0.2.2",
-    "localhost"
+    os.getenv('ALLOWED_HOSTS')
```

Comment:
do we get array vals In env? In node, it usually comes as a string.

Reply:
My bad; let me add the correction